### ISSUE TITLE
How to use IOrderableCustomEntityDefinition

### ISSUE THREAD

**Issue by ArcaneTSGK** [cofoundry-cms/cofoundry#370]
Hi,

I am using CoFoundry as a headless CMS and use an API to query data, one of those data sets is a list of Promotion pages which include promotions that are specific to different categories and days of the week. I would like to give full control to the marketing team to order these pages however they choose, so that when the front end site is rebuilt the promotions are ordered how the API returns them, rather than having to sanitize the data on the front end.

I have a custom entity definition like the following

```
public class PromotionDetailCustomEntityDefinition: ICustomEntityDefinition<PromotionDetailDataModel>, IOrderableCustomEntityDefinition, ISortedCustomEntityDefinition
{
  ...
  public CustomEntityOrdering ordering => customEntityOrdering.Full;

  public CustomEntityQuerySortType DefaultSortType => CustomEntityQuerySortType.Natural;

  public SortDirection DefaultSortDirection => SortDirection.Default;
}
```

How do I implement the ordering, In the Admin UI I can see a button 'Change ordering' but when I click it nothing happens. 

Ideally I want to be able to just drag and drop the order of the entities in there if that's at all possible, so that when the API is called it returns the pages as they have been ordered, however if it has to be done by code and ordered based on some field within each custom entity, then I don't mind this either, I am just finding it difficult to find an example on how to actually use the ordering.

Thanks

**Comment by HeyJoel** [comment]
Hi @ArcaneTSGK, there was a bug with the admin panel that was preventing the ordering dialog from opening. I've fixed it, and it will be release in v0.7 which should be out in the next week or two.

If you need the fix urgently, then [this pre-release package on MyGet](https://myget.org/feed/cofoundry/package/nuget/Cofoundry.Web.Admin/0.6.2-ci0014) will contain the bug fix (without the v0.7 breaking changes).

Note that `CustomEntityOrdering.Full` is only intended to support short lists of up to 60 items, whereafter it gets impractical to manual sort such a large number of items and partial ordering is more practical.

**Comment by HeyJoel** [comment]
Fixed in 0.7.0 which is now released.
